### ISSUE TITLE
Can't access folder if folder name ends with space

### ISSUE THREAD

**Issue by InnerPeaceKR** [kapitainsky/RcloneBrowser#107]
Can't access folder if folder name ends with space such as "12345 "

![1](https://user-images.githubusercontent.com/11524657/80111779-4880ca80-85bb-11ea-9821-9f3e3cfd18ce.png)

**Comment by kapitainsky** [comment]
what remote you are using?

**Comment by kapitainsky** [comment]
try from command line to check if it is RB or rclone problem

`rclone size "remote:test/abc "`

**Comment by InnerPeaceKR** [comment]
Remote is google drive and

C:\rclone>rclone size "gdrive:/test/12345 "
Total objects: 1
Total size: 26 Bytes (26 Bytes)

C:\rclone>

Works fine.

**Comment by kapitainsky** [comment]
Ok will look into it but it is not good practice to use file names like this - on any system. 

Quick Google tells me that many tools can't handle it properly. It is edge case better to be avoided.

**Comment by InnerPeaceKR** [comment]
Absolutely. I never make a folder like that. I just tried to access shared folder via RB and got a problem.

**Comment by kapitainsky** [comment]
it is impossible to fix at the moment as rclone does not handle such names properly.

you can create folder like this:

`rclone mkdir "google:test "`

but then when you try to list remote content this is rclone output:

```
$ rclone lsd google:
          -1 2020-03-03 21:09:16        -1 My Music
          -1 2020-04-22 12:55:44        -1 My Pictures
          -1 2020-04-21 07:23:24        -1 My Videos
          -1 2020-04-23 16:08:01        -1 test
```
I checked- there is no space after test. RB uses rclone to do all remotes operations. If there are trailing spaces it does not see them

**Comment by kapitainsky** [comment]
But will try to investigate further when I have a moment. Would be nice to handle it.

Thank you for reporting. Hopefully this issue affects very few people

**Comment by kapitainsky** [comment]
I've found the way to fix it - but have mix feelings whether to do this or not... I can imagine people downloading files like these to their system and then struggling to even delete them -  example [here](https://community.spiceworks.com/topic/273265-files-with-spaces-on-the-end-can-t-delete-files). I will let it be for time being when I consider what is the best approach.

**Comment by kapitainsky** [comment]
At the same time I have no intention to create tool holding end user hand and preventing any possible mistakes or problems. It is a tool not space rocket controller. Any thoughts?

**Comment by InnerPeaceKR** [comment]
I found a similar issue on Rclone page.
https://github.com/rclone/rclone/issues/2333
I think that it would be enough to mark as known issue.

**Comment by kapitainsky** [comment]
I will enable support of files with leading/trailing spaces in RB but will not allow creation of such files (mkdir).

**Comment by kapitainsky** [comment]
I did a bit of investigation and it seems that all major OS (macOS, Linux and Windows) do support leading and trailing spaces. Hence if also supported by remote I don't see any reason not to support it fully too in RB.
